packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -61,7 +62,7 @@ export const DatePicker = (inProps: DatePickerProps) => {
                         {endAdornment}
                     </>
                 ) : (
-                    endAdornment
+                    { endAdornment }
```

Comment:
Is this necessary? 
If this was broken before we should add a `patch` changeset mentioning the fix.